TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -84,15 +85,25 @@ public TrafficReplayer(URI serverUri,
                            IAuthTransformerFactory authTransformerFactory,
                            boolean allowInsecureConnections)
             throws SSLException {
-        this(serverUri, authTransformerFactory, allowInsecureConnections, 1,
+        this(serverUri, authTransformerFactory, allowInsecureConnections, 0, 1024);
```

Comment:
Why is numSendingThreads 0 here?

Reply:
0 tells netty to decide.  It will decide based upon the number of CPU cores that are on the host.